static/css/_task-template.scss

```diff
@@ -71,6 +71,18 @@
     }
 }
 
+._file-upload {
+  & .md-button {
+    margin-bottom: 20px;
+  }
+  & #progress {
```

Comment:
just a hint, if two css rules are exactly the same then you do: `rule1, rule2 { margin-top:15 px;}`